engine/src/aldrv/al_init.cpp

```diff
@@ -135,7 +159,7 @@ static ALCcontext *context_id = NULL;
 #endif
 bool AUDInit()
 {
-    g_game.sound_enabled = 
+    g_game.sound_enabled =
```

Comment:
Q: What is this suppose to equal?
I see it's on both sides just removing a space, but I don't see an rvalue, and if HAVE_AL is undefined then there won't be an rvalue either (otherwise the rvalue would be the value of `usedoppler`.

Seems like a bug.

Reply:
There have been unexplained sound failures for ages this maybe the source

Reply:
let's at least file an issue so we can resolve this. I'll look later on to see if there are others that pop out from this too.